packages/patternfly-4/react-core/src/components/Accordion/Accordion.tsx

```diff
@@ -2,22 +2,44 @@ import * as React from 'react';
 import { css } from '@patternfly/react-styles';
 import styles from '@patternfly/react-styles/css/components/Accordion/accordion';
 
+export const AccordionContext = React.createContext('h3');
+
+export enum HeadingLevel {
```

Comment:
Judging by our discussion in Slack, since the user already has `TitleLevel` we don't need this enum after all.